Makefile

```diff
@@ -136,6 +137,16 @@ $(TEST): $(SOURCE) $(SPECS) $(BIN)
 testem:
 	$(BIN)/testem
 
+# Rules to attach checksums to JavaScript source in the header
+app.html: app.html.in .make-script-block
+	sed -e '/%SCRIPTS%/ {' -e 'r .make-script-block' -e 'd' -e '}' app.html.in > $(DIST)/app.html
```

Comment:
sorry. i know i asked you to put the app.html in DIST but for this PR can you ignore what i said and just make it in the base directory.

we'll need to make more changes in order to use the dist dir, so lets wait and define that as a separate task.